### PR TITLE
[TechDraw] Translations and grammer

### PR DESCRIPTION
**Pull request by benj5378**

### CODE REVIEW

File: src/Mod/TechDraw/App/DrawViewDimension.cpp [hunk 1]
```diff
@@ -1154,7 +1154,7 @@ arcPoints DrawViewDimension::arcPointsFromEdge(TopoDS_Edge occEdge)
             pts.isArc = isArc;
             BRepAdaptor_Curve adaptCircle(circleEdge);
             if (adaptCircle.GetType() != GeomAbs_Circle) {
-                throw Base::RuntimeError("failed to get circle from bspline");
+                throw Base::RuntimeError("failed to get circle from bB-pline");
```
Comment (Reqrefusion) [inline src/Mod/TechDraw/App/DrawViewDimension.cpp:1157]: You forget lowercase b

**Comment by luzpaz** [comment]
CC @kaktusus (thought this would be informative from a translation perspective)

**Comment by kaktusus** [comment]
**I will look into it.**

**Comment by kaktusus** [comment]
The proposed changes to the strings are very matter-of-fact.
I am glad that **_benj5378_** found time to implement them.


This workbench needs more attention - see: https://github.com/FreeCAD/FreeCAD-translations/issues?q=+is%3Aopen+TechDraw+

looking at the notation:
`<string notr="true">ANSI</string>`
For me there is a space missing _(for better readability)_, I hope it will work.
`<string notr="true" >ANSI</string>` 
Fortunately, I am not an expert in this field. :wink:

**Comment by maxwxyz** [comment]
@benj5378 could you please resolve the conflicts? TIA

**Comment by maxwxyz** [comment]
Looked at this at the merge meeting but it still has conflicts. Could you please resolve them?

**Pull request merged**